api/graphql/schema.graphql

```diff
@@ -518,14 +518,22 @@ type Department {
 
 type Team {
   id: ID!
-  name: String
+  name: String!
```

Comment:
Since we're going to have confusion around this one but can't change it, could you add a comment explaining how it is a unique key for the team?

Reply:
sure 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/5694/commits/13c1c812977373a92534181ac78b6268e89a1362